src/app/containers/PageHandlers/withData.jsx

```diff
@@ -1,31 +1,46 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { shape, element } from 'prop-types';
 import articlePropTypes from '../../models/propTypes/article';
 import ErrorMain from '../ErrorMain';
 import nodeLogger from '../../helpers/logger.node';
 import deepGet from '../../helpers/json/deepGet';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import { getArticleHomeService } from '../PageViewAnalytics/labelHelpers/article';
 
 const logger = nodeLogger(__filename);
 
-// checks for data and status keys, setting default status if not found
+// checks for data, status & home keys, setting default status if not found
 const constructRenderObject = data => ({
   status: deepGet(['status'], data) || 500,
   pageData: deepGet(['pageData'], data),
+  home: deepGet(['pageData', 'metadata', 'passport', 'home'], data),
 });
 
-// checks for pageData and 200 status
+// checks for pageData, 200 status and if home service from article data fits the service locale
 const shouldRender = data => {
   const { status, pageData } = constructRenderObject(data);
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const { service } = useContext(ServiceContext);
 
   const hasDataAnd200Status = pageData && status === 200;
+  const homeService = pageData && getArticleHomeService(pageData);
 
-  return { hasDataAnd200Status, status, pageData };
+  const isValidHome = pageData && homeService === service;
```

Comment:
Renaming instances of `homeService` to `passportHome` would be helpful, since it is the home field in the Passport of the data. 

There's something to note: only Article pages currently have a passport home field - the World Service front pages do not yet have this field.  So there's an extra case to test.  If `passportHome` returns null, the page should not 404. 
It should only 404 if `passportHome` is defined and is different to the `service`.